### ISSUE TITLE
Enhance focus indicators

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5145]
## Type of report

Feature request

## Provide description of the new feature

At the moment the only focus indicator for the editor is the text cursor. However, such an indication won't work in cases when the cursor is not visible (e.g. read-only editors). We should add some other focus indication, e.g. outline to the whole editor's area.
